src/iexamine.cpp

```diff
@@ -2609,10 +2615,10 @@ void iexamine::fireplace( player &p, const tripoint &examp )
         if( has_bionic_firestarter ) {
             selection_menu.addentry( 2, true, 'b', _( "Use a CBM to start a fire" ) );
         }
-    } else if( p.weapon.damage_melee( DT_BASH ) ) {
+    } else if( has_firequencher ) {
         selection_menu.addentry( 4, true, 's', _( "Put out fire" ) );
     } else {
-        selection_menu.addentry( 4, false, 's', _( "Put out fire - wield bashing item" ) );
+        selection_menu.addentry( 4, false, 's', _( "Put out fire - obtain a bashing item" ) );
```

Comment:
Don't know if you like this variant. @nexusmrsep didn't like it.
```suggestion
        selection_menu.addentry( 4, false, 's', _( "Put out fire (bashing item required)" ) );
```